datafusion/src/physical_plan/union.rs

```diff
@@ -43,7 +43,7 @@ use async_trait::async_trait;
 #[derive(Debug)]
 pub struct UnionExec {
     /// Input execution plan
-    inputs: Vec<Arc<dyn ExecutionPlan>>,
+    pub inputs: Vec<Arc<dyn ExecutionPlan>>,
```

Comment:
better keep it consistent across different physical plans, other plans keep this field private and choose to expose it through an `input` method instead, see the limit plan for an example.

Reply:
Thanks ❤️ fixed.